src/components/contingency-list.js

```diff
@@ -649,6 +653,20 @@ const ContingencyLists = () => {
                         {newListCreated && (
                             <NewFileCreatedList>
                                 <CustomListItem button selected>
+                                    <div
+                                        style={{
+                                            marginLeft: '5px',
```

Comment:
I think, we must apply the same css of the final list to have the same rendering for the temporary list:
marginLeft: '10px', 
marginRight: 15px '
![temporary-list](https://user-images.githubusercontent.com/13727922/102057132-bbaaab80-3ded-11eb-9e6c-1a1148abf10b.png)